### PR TITLE
fix(dropdown): keyboard navigation

### LINKED ISSUE #1228: Arrow keys don't work in DropdownToggle
In the dropdown toggle, the arrow keys to navigate through the items in the list don't work.

Reproducable by going to http://angular-ui.github.io/bootstrap/#/dropdownToggle and selecting "Edit in plunker".

### PR DESCRIPTION
**Pull request by yarl**
Add option to use up and down arrows to navigate through list.
Fixes #1228

### CODE REVIEW

**Comment by karianna** [comment]
Hi @yarl - Thanks for the PR - are you also able to add some test(s) around it?

**Comment by porjo** [comment]
+1

File: src/dropdown/dropdown.js [hunk 1]
```diff
@@ -90,6 +91,27 @@ angular.module('ui.bootstrap.dropdown', [])
       self.toggleElement[0].focus();
     }
   };
+  
+  scope.selectOption = function( evt ) {
+    var elems = angular.element(self.$element).find('.dropdown-menu a');
```
Comment (wesleycho) [inline src/dropdown/dropdown.js]: It would be better to create a directive where when a value is true, then it focuses on the element.

**Comment by wesleycho** [comment]
As mentioned, this is in need of tests currently.

**Comment by bleggett** [comment]
[Have a PR] (https://github.com/angular-ui/bootstrap/pull/3685) that adds this functionality. Started with yarl's fix but made it an optional flag, added some tests, and removed the apparent dependency on jQuery.

**Pull request closed without merging**